runtime/near-vm-runner/src/cache.rs

```diff
@@ -235,11 +235,16 @@ struct FilesystemContractRuntimeCacheState {
 
 #[cfg(not(windows))]
 impl FilesystemContractRuntimeCache {
-    pub fn new<SP: AsRef<std::path::Path> + ?Sized>(
+    pub fn new<SP, CCP>(
```

Comment:
nit: Can you rename to something meaningful like StorePath and ContractCachePath?

Reply:
also does it need to be this fancy? Can you we just use Path directly?